### DISCUSSION TITLE
Sonoff USB dongle 3.0 plus E non riconosciuta da Home Assistant

### DISCUSSION THREAD

**Discussion by MarkTam-IoT** [indomus/forum#2915] [Hardware & Firmware]
Buon pomeriggio, 
dopo aver seguito le varie guide sono riuscito ad installare home assitant. Al primo avvio pero' vede gli altri dispositivi ma non il dongle USB di Sonoff. Anche il successivo tentativo di integrazione del dispositivo ZHA attraverso linterfaccia di HASS non va a buon fine: l'inserimento manuale della porta a cui si troverebbe il dispositivo non produce il  risultato sperato. 
Avete consigli??
Grazie

**Comment by snakuzzo** [comment]
No, se prima non dettagli meglio che installazione hai

**Reply by MarkTam-IoT** [reply]
Home assistant core su docker con raspberry pi. L'integrazione di ZHA con radio EZP e inserimento di  /dev/serial/by-id/usb-ITEAD_SONOFF_Zigbee_3.0_USB_Dongle_Plus_V2_20231101132951-if00 nelle impostazioni della porta seriale, con controllo flusso dati software non risolve il problema, il dispositivo continua a non connettersi

**Reply by snakuzzo** [reply]
Hai fatto il bind del device quando hai creato il container ?

**Reply by MarkTam-IoT** [reply]
magari e' un passaggio che mi manca....potresti darmi indicazioni??

**Reply by snakuzzo** [reply]
Leggi qui... https://indomus.it/guide/zha-zigbee-home-automation-come-configurarlo-su-home-assistant/

Paragrafo "Su Home Assistant Core"

**Comment by MarkTam-IoT** [comment]
buongiorno, quando eseguo le istruzioni per il reinstanziamento con il comando _docker compose up -d home-assistant_ ottengo il seguente errore
pi@raspberrypi:~ $ docker compose up -d home-assistant
unknown shorthand flag: 'd' in -d
See 'docker --help'.
con il trattino invece, come suggerito nella guida ottengo il seguente errore:
pi@raspberrypi:~ $ docker-compose up -d home-assistant
-bash: docker-compose: command not found
dove sbaglio??
Grazie

**Reply by snakuzzo** [reply]
Domande:
Hai istanziato Core con docker o con docker compose ?
Qui è spiegata la differenza tra `docker compose` e `docker-compose` -> https://indomus.it/formazione/gestire-applicazioni-docker-tramite-docker-compose/

**Reply by MarkTam-IoT** [reply]
Con docker compose, ma mi viene il dubbio sul file docker-compose.yaml. Dove va creato? In quale directory?

**Reply by snakuzzo** [reply]
Spiegami cosa stai combinando.
Tu il container di core già lo avevi...io ti ho dato una semplice indicazione sul discorso del bind device.

**Reply by MarkTam-IoT** [reply]
Ho rieffettuato l'installazione di di raspbian e docker per poi proseguire con il docker compose per home assistant. Non mi e' chiaro dove deve essere creato il file docker-compose.yaml

**Reply by snakuzzo** [reply]
Ma scusa eh...se già è up&running, come fai a non sapere dov'è il docker-compose.yaml?

**Reply by snakuzzo** [reply]
Hai detto che già avevi core, come l'hai installato?

**Reply by MarkTam-IoT** [reply]
Ho risolto, grazie per la tua disponibilita'
